agent-install/agent-install.sh

```diff
@@ -147,7 +153,7 @@ function now() {
 
 # Get cmd line args. Each arg corresponds to an env var. The arg value will be stored in ARG_<envvar>. Then the get_variable function will choose the highest precedence variable set.
 # Note: could not put this in a function, because getopts would only process the function args
-AGENT_VERBOSITY=3   # default until we get it from all of the possible places
+AGENT_VERBOSITY=5   # default until we get it from all of the possible places
```

Comment:
Was this changed for your testing, or is it defaulting to 5 now?

Reply:
Yes, I will switch it back to 3.